metrics/utils.go

```diff
@@ -31,6 +31,7 @@ func measuredMethod(m string) string {
 		"HEAD",
 		"POST",
 		"PUT",
+		"PATCH",
```

Comment:
Thank you for the fix. Could please also add TRACE and CONNECT methods such that we end up having all methods from here https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods